frontend/packages/console-app/src/components/cloud-shell/__tests__/useCloudShellAvailable.spec.ts

```diff
@@ -0,0 +1,51 @@
+import { act } from 'react-dom/test-utils';
+import useCloudShellAvailable from '../useCloudShellAvailable';
+import { checkTerminalAvailable } from '../cloud-shell-utils';
+import { useFlag } from '@console/shared';
+import { testHook } from '../../../../../../__tests__/utils/hooks-utils';
+
+const useFlagMock = useFlag as jest.Mock;
+const checkTerminalAvailableMock = checkTerminalAvailable as jest.Mock;
+
+jest.mock('../cloud-shell-utils', () => {
+  return {
+    checkTerminalAvailable: jest.fn(() => Promise.resolve()),
+  };
+});
+
+jest.mock('@console/shared', () => {
+  const originalModule = (jest as any).requireActual('@console/shared');
+  return {
+    ...originalModule,
+    useFlag: jest.fn(() => true),
```

Comment:
I'm flip-flopping on making this comment... but I think it's better to not have a default for your methods. 

You're mocking them when you need to specify a sane test (all parts included). So if you have no default returns for these methods, a lack of a mock should fail due to no explicit definition of the return type (it'll get back `undefined` or never resolve)... which I think helps make the tests whole, if more are written in the future.